CHANGELOG.rst

```diff
@@ -16,80 +53,43 @@ in development
   Also add two new corresponding CLI commands - ``st2 alias-execution match`` and
   ``st2 alias-execution execute``. (new feature) #2895
 
-  Contributed by Anthony Shaw
-* Only allow valid word characters (``a-z``, ``0-9`` and ``_``) to be used for action parameter
-  names. Previously, due to bug in the code, any character was allowed.
+  Contribution by Anthony Shaw
```

Comment:
Pls fix formatting: not clear which line "contribution" belongs to - above or below.